docs/changes.xml

```diff
@@ -26,6 +26,15 @@ NGINX Unit updated to 1.32.0.
 </para>
 </change>
 
+<change type="bugfix">
+<para>
+Unit's Node.js replacement http package brought the signature of http.createServer
+function and http.Server constructors in line with modern Node.js to include an option
+object to allow modern applications to function. Unit discards the passed in option
+object and prints to log that it has done so.
+</para>
```

Comment:
Changes are just a description for users who briefly scanning release notes to find something related to them. So short note with keywords should be enough. Consider following:

`http.createServer() now accepts "options" argument introduced in Node.js v9.6.0, v8.12.0.`

Reply:
This has now been changed to this version, thank you!